lib/presentation/view/modal_sheets/edit_text_modal_sheet_builder.dart

```diff
@@ -165,21 +170,23 @@ class EditTextModalSheetBuilder {
                         mainAxisAlignment: MainAxisAlignment.end,
                         children: [
                           TextButton(
-                            onPressed: () => {_appNavigation.popBack(), _debounce?.cancel()},
+                            onPressed: () => {Navigator.pop(context), _debounce?.cancel(), _onCancelActionClick()},
```

Comment:
why change from `_appNavigation.popBack()` to `Navigator.pop(context)`

Reply:
Because the currentState's RouteEntry list does not contain the RouteEntry generated by BottomSheet when opening BottomSheet. So currentState.canPop () always returns false